routes/orders.js

```diff
@@ -9,6 +9,7 @@ const {
     createOrder, 
     getOrdersByUser
 } = require('../db')
+const { user } = require('../db/client')
```

Comment:
REQUIRED - First check if this is being used somewhere, then if not, delete.

Reply:
Stop yelling at me.